config/routes.rb

```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
-  resources :items, only: [:index]
+  resources :items, only: [:index, :show]
   resources :categories, only: [:show], param: :slug
   resources :users, only: [:index]
+  resources :cart_items, only: [:create]
+  # resource :cart, only: [:show]
+
+  get '/cart', to: 'cart_items#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.